### ISSUE TITLE
Hot Gym Tutorial- ./swarm.py: Permission denied 

### ISSUE THREAD

**Issue by camer1kj** [numenta/nupic#2236]
Trying to run the hot gym tutorial and it wouldn't let me execute the swarm.py file. (-bash: ./swarm.py: Permission denied)
After I manually made it executable, I am getting an error that there is no such file or directory 
(IOError: [Errno 2] No such file or directory) concerning nupic-default.xml.

**Comment by camer1kj** [comment]
It was true that while most of the other files were in that spot, nupic-default.xml was missing. After manually copying it into the correct directory from a colleague's computer, the program got hung up on another missing file 
(IOError: [Errno 2] No such file or directory: '/home/kjcamero/.local/lib/python2.7/site-packages/nupic/frameworks/opf/exp_generator/experimentDescriptionSchema.json)

Any ideas why so many of the files are missing?

**Comment by breznak** [comment]
@camer1kj looks like your installation didn't go 100%. Can you try clone the files yourself from git? or download the zip archive with latest `master` from here on Github. 
Is the issue still valid, or can we close?

**Comment by rhyolight** [comment]
@camer1kj Yes, looks like a bad install. How did you install NuPIC?

**Comment by camer1kj** [comment]
Yes, these errors came from the pip install. After uninstalling then downloading from source it worked fine.

**Comment by rhyolight** [comment]
Hm... what OS are you using? If Linux, then "pip install nupic" does not
work.

---------
Matt Taylor
OS Community Flag-Bearer
Numenta

**Comment by camer1kj** [comment]
Yes I am running on Linux. But I used the wheel download link from the nupic website: 
pip install https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic/releases/nupic-0.2.5-cp27-none-linux_x86_64.whl
This is when I was missing files.

**Comment by rhyolight** [comment]
Damn, that should have worked! Do you happen to be on a 32b system?

---------
Matt Taylor
OS Community Flag-Bearer
Numenta

**Comment by camer1kj** [comment]
Nope, 64bit
